cmd/utils/flags.go

```diff
@@ -138,6 +138,10 @@ var (
 		Name:  "garden",
 		Usage: "Garden network: pre-configured proof-of-work test network",
 	}
+	LocalFlag = cli.BoolFlag{
+		Name:  "local",
+		Usage: "Local network: pre-configured proof-of-work test network",
```

Comment:
(nit) probably worth mentioning this is an offline only mode of operation

Reply:
you need this if you want to have a flag for local. Did you just refer to the `Usage` comment?

Reply:
Yeah, just the usage comment. It would be good if it explained the command a bit, haha